src/components/work/details/Details.js

```diff
@@ -63,11 +63,18 @@ function Details({
     return [...new Set(mixed)];
   }, [manifestation]);
 
+  const materialType = manifestation?.materialTypes?.[0]?.specific;
```

Comment:
Jeg tror det skal være alle MaterialTyperne på Manifestationerne...?

Reply:
udover det skal vi nok bruge mostRelevant i vores kald istedet  for all